xdevel.sh

```diff
@@ -6,6 +6,6 @@ system_requirement_missing: |
    * On Ubuntu-compatible systems you probably need: libxpm-dev libxext-dev libx11-dev libxft-dev
 system_requirement: ".*"
 system_requirement_check: |
-  printf "#include <X11/Xlib.h>\n#include <X11/xpm.h>\n#include <X11/Xft/Xft.h>\n#include <X11/extensions/Xext.h>\n" | cc -xc - -I/opt/X11/include `pkg-config freetype2 --cflags` -c -o /dev/null
+  printf "#include <X11/Xlib.h>\n#include <X11/xpm.h>\n#include <X11/Xft/Xft.h>\n#include <X11/extensions/Xext.h>\n" | cc -xc - -I/opt/X11/include $(freetype-config --cflags 2> /dev/null && pkg-config freetype2 --cflags 2> /dev/null)  -c -o /dev/null
```

Comment:
$(freetype-config --cflags 2> /dev/null && pkg-config freetype2 --cflags 2> /dev/null)
probably should be
$(freetype-config --cflags 2> /dev/null || pkg-config freetype2 --cflags 2> /dev/null)